client/asset/interface.go

```diff
@@ -1017,6 +1027,47 @@ type TicketBuyer interface {
 	TicketPage(scanStart int32, n, skipN int) ([]*Ticket, error)
 }
 
+// TransactionType is type type of transaction made by a wallet.
+type TransactionType uint16
+
+const (
+	Send TransactionType = iota
+	Receive
+	Swap
+	Redeem
+	Refund
+	CreateBond
+	RedeemBond
+	ApproveToken
+	Acceleration
+)
```

Comment:
0 as type Unknown just in case we need it? dcr also might have a `Split` type and a set of stake types. The treasury types...

Reply:
I'll add an `Unknown`. There's definitely a good amount missing. This can keep getting updated as the various wallets are added.